app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -908,14 +929,15 @@ void
 BrowserWorld::DrawWorld() {
   m.externalVR->SetCompositorEnabled(true);
   m.device->SetRenderMode(device::RenderMode::StandAlone);
+  if (m.fadeAnimation) {
+    m.fadeAnimation->UpdateAnimation();
```

Comment:
m.fadeAnimation seems like a good candidate for vrb::Updatable, that way you can use RenderContext::GetFrameDelta() to make the animations time based. If you agree, just file as a follow up.